aws/eks-anywhere
# Add unit test to verify generated bundle against golden file

Add unit test to verify generated bundle against golden file
Following changes were done to make the bundle file reproducible in dry-run
* Set predictable fake SHA for image and archives
* Set release time to Unix epoch start time 

The test will run the dry-run of dev-release on both `main` branch and release branches.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
